src/analyzer/analyzerbeats.cpp

```diff
@@ -158,7 +158,9 @@ bool AnalyzerBeats::shouldAnalyze(TrackPointer pTrack) const {
         return m_bPreferencesReanalyzeImported;
     }
 
-    if (subVersion.isEmpty() && pBeats->findNextBeat(0) <= 0.0 &&
+    if (subVersion.isEmpty() &&
+            pBeats->findNextBeat(mixxx::audio::kStartFramePos) <=
```

Comment:
How about introducing a utility function to reduce the cognitive load?

```
Beats::findFirstBeat() = Beats::findNextBeat(mixxx::audio::kStartFramePos)
```

Reply:
469c9908f3c3de4b307a332c13de9705c3db9579